### ISSUE TITLE
What type of loss did you use during training?

### ISSUE THREAD

**Issue by Le1Elayne** [sniklaus/pytorch-pwc#28]
I try to train this network using multiscale loss described in the paper. But it failed. It works when I use L2 loss. So what type of loss did you use during training? Could you please explain why it does not work with multiscale loss? Thank you!

**Comment by sniklaus** [comment]
This question might be better suited for the official repository. Please kindly reach out to the authors there: https://github.com/NVlabs/PWC-Net
